Week_6/homework/backend/routes/todo.js

```diff
@@ -0,0 +1,55 @@
+/** Module for handling users */
+const express = require("express");
+const cors = require("cors");
+
+// Define route and middlewares
+const todo = express.Router();
+todo.use(cors());
+todo.use(express.json());
+
+  todo.post("/", async (req, res) => {
+    const todos = db.collection("todos");
+    const body = req.body;
+
+    const doc = await db.collection('todos').add({
+      todo: body.todo,
+      email: body.email,
+    });
+
+    const doc2 = await db.collection('todos').doc(doc.id).update({
```

Comment:
Alternatively, you don't have to make two calls to the database to augment the todo (since you're duplicating the data).

Just save doc  and in the /GET route, augment the data with the uid then:
```todos.docs.map(obj => {const data = {...obj.data(), uid: obj.id}; return data})```